cmd/cdi-controller/controller.go

```diff
@@ -409,6 +410,9 @@ func getCacheOptions(apiClient client.Client, cdiNamespace string) cache.Options
 			&batchv1.Job{}: {
 				Field: namespaceSelector,
 			},
+			&v1.ConfigMap{}: {
+				Field: namespaceSelector,
+			},
```

Comment:
I think the same treatment may be needed for Secrets, or, could use uncached client for both instead and drop this, since we're only ever going to GET on those (which should be cheap).
@mhenriks WDYT?

Reply:
ping @awels @mhenriks